modules/rosa-planning-considerations.adoc

```diff
@@ -32,11 +35,53 @@ The following table lists the size of control plane and infrastructure nodes tha
 |===
 [.small]
 --
-1. The maximum number of worker nodes on ROSA is 180
+1. The maximum number of compute nodes on ROSA is 180.
 --
 
-For larger clusters, infrastructure node sizing can become a large impacting factor to scalability. There are many factors that influence the stated thresholds, including the etcd version or storage data format.
+[id="node-scaling-after-installation_{context}"]
+== Node scaling after installation
 
-Exceeding these limits does not necessarily mean that the cluster will fail. In most cases, exceeding these numbers results in lower overall performance.
+If you change the number of compute nodes after installation, the control plane and infrastructure nodes are scaled by the Red Hat Site Reliability Engineering (SRE) team as required. The nodes are scaled to maintain platform stability.
+
+Post-installation scaling requirements for control plane and infrastructure nodes are assessed on a case-by-case basis. Node resource consumption and received alerts are taken into consideration.
+
+The following table details the alerting rules that define when resizing alerts are triggered for a cluster. 
+
+[options="header",cols="1,1,2"]
+|===
+|Node type |Operand |Conditions
+
+.3+|Control plane node scaling
+
+.3+|`OR`
+
+|Each control plane node has less than 16GiB RAM, and there are more than 25 and less than 101 compute nodes.
+|Each control plane node has less than 32GiB RAM, and there are more than 100 and less than 251 compute nodes.
+|Each control plane node has less than 64GiB RAM and there are more than 250 compute nodes.
+
+.4+|Infrastructure node scaling
 
-The OpenShift Container Platform version used in all of the tests is OCP 4.8.0.
+.4+|`OR`
+
+|Each infrastructure node has less than 16GiB RAM or less than 5 CPUs, and there are more than 25 and less than 101 compute nodes.
+|Each infrastructure node has less than 32GiB RAM or less than 9 CPUs, and there are more than 100 and less than 251 compute nodes.
+|Each infrastructure node has less than 128GiB RAM or less than 17 CPUs, and there are more than 250 compute nodes.
+|Each infrastructure node has less than 128GiB RAM or less than 33 CPUs, and there are more than 500 compute nodes.
+
+|===
+
+[NOTE]
+====
+For more information about the alerting rules that trigger control plane and infrastructure node resizing alerts, see the link:https://github.com/openshift/managed-cluster-config/blob/master/deploy/sre-prometheus/100-control-plane-resizing.PrometheusRule.yaml[Prometheus rule configuration file] in the managed-cluster-config repository on GitHub.
+====
+
+The SRE team might scale the control plane and infrastructure nodes for additional reasons, for example to manage an increase in resource consumption on the nodes.
+
+When scaling is applied, the customer is notified through a service log entry.
```

Comment:
A link to more info on "service log" here will be useful.

Reply:
Thank you for this suggestion, this is a great idea. On review, I can't find a relevant section on this topic in the ROSA documentation. I have subsequently created https://issues.redhat.com/browse/OSDOCS-3718 to track this docs requirement outside of this PR.

Reply:
A page about accessing the service log for ROSA is added in https://github.com/openshift/openshift-docs/pull/47033. I have also included a cross-reference in the PR under this section in the scaling content.